### DISCUSSION TITLE
4/14 - Mapping Decolonized Futures: Indigenous Visions for Palestine and Hawai’i

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#8] [Extra Credit]
### 10 points

# Prompt
>    ##  How is the countermap practice connected to critical cartography? How might you apply it in your own map making?

Attend the [Mapping Decolonized Futures: Indigenous Visions for Palestine and Hawai’i](https://asianam.ucla.edu/event/research-talk-series-mapping-decolonized-futures-indigenous-visions-for-palestine-and-hawaii/) talk on Friday, April 14th from 2pm - 3:30pm and reflect in at least 2-3 sentences on the prompt above.

**Comment by emai14** [comment]
The counter-map practice and critical cartography have the same goal of decolonizing mapmaking. The counter-map practice that Dr. Joudah discussed involved pre-colonial placenames and spaces, and a collection of indigenous history within a map, in an effort to reclaim colonized territory. In our own maps, we can also work to place and acknowledge the importance of indigenous knowledge, and counter the dominant western hegemonies which inform traditional mapping in the western world. In a similar way, we can also acknowledge not only the stealing of land, but also the process of gentrification through countermapping.

**Reply by cassidytu** [reply]
Dr. Joudah's quote, "looking for yourself in a place that denies you" and her emphasis on how these projects are made by and for the communities were a few of the things I remembered the most in response to the dominant western hegemonies and gentrification you mentioned. I like how you connected the talk to how we can personally implement these ideas of counter-mapping into our own maps, especially since the talk got pretty theoretical (making it bit difficult for me to fully understand).

**Comment by cassidytu** [comment]
Counter-mapping is utilizing mapping as a form of activism against dominant power structures, which ties in to critical cartography that analyzes how maps reflect and further perpetuate relations of these dominant powers. Dr. Joudah highlights several counter-mapping initiatives that combat the settler colonial temporal trap, by interrupting the current narrative and redefining terms to reclaim ownership. The projects range from mapathons ([Palestine Open Maps](https://palopenmaps.org/)) to archiving ([Papakilo Database](https://www.papakilodatabase.com/)) to designing the reconstruction of destroyed villages ([Palestine Land Society Reconstruction Competition](https://www.plands.org/en/competition-news/competition-history-and-operation-year-6); my favorite part!).

To anyone who wants to watch the talk, here's a recording of it that Dr. Joudah did at another school: 
[Nour Joudah: Mapping Decolonized Futures](https://www.youtube.com/watch?v=LOd2-1ojXyA&t=7s)
